server.js

```diff
@@ -37,39 +42,49 @@ app.get('/throw-an-error', (request, response) => {
   throw 'Something went really wrong!';
 });
 
+
 //Weather Route for json data
-async function getWeatherDaily(request, response) {
-  // try 
-  let cityName = request.query.city_name;
-  let latitude = request.query.lat;
-  let longitude = request.query.lon;
-
-  let url = `https://api.weatherbit.io/v2.0/forecast/daily?lat=${latitude}&lon=${longitude}&key=${process.env.WEATHER_API_KEY}`;
-
-  try {
-    let weatherResults = await axios.get(url);
-    let WeatherArr = weatherResults.data.data.map(day => new Forecast(day));
-    response.send(WeatherArr);
-  } catch (error) {response.send('error');
-  }
+async function getWeather(request, response) {
+  let lat = request.query.latt;
+  let lon = request.query.lon;
+
+  let url = `https://api.weatherbit.io/v2.0/forecast/daily?&lat=${lat}&lon=${lon}&key=${process.env.WEATHER_API_KEY}&units=I`;
+
+  let weatherResults = await axios.get(url);
+  let WeatherArr = weatherResults.data.data.map(day => new Forecast(day));
```

Comment:
- We are using uppercase `WeatherArr` but we want to use camelCase. Update to `weatherArr` 👍